src/Cms/Language.php

```diff
@@ -517,6 +533,16 @@ public function slugs(): array
     {
         return $this->slugs;
     }
+    
+    /**
+     * Returns the custom smartypants for this language
```

Comment:
I think this should be "custom SmartyPants *options*" to make clearer what the property does.